### PR TITLE
fix add react-native-get-random-values library

### PR DESCRIPTION
**Pull request by kaushalkapasi**
- support uuid random number generation in react-native

### CODE REVIEW

File: sdk/js/package.json [hunk 1]
```diff
@@ -3,5 +3,8 @@
   "version": "1.2.9",
   "description": "The DevCycle JS SDK used for feature management.",
   "author": "",
-  "license": "MIT"
+  "license": "MIT",
+  "dependencies": {
+    "react-native-get-random-values": "^1.8.0"
```
Comment (ajwootto) [inline sdk/js/package.json:8]: this should be added to the root package.json, not here

File: sdk/js/src/User.ts [hunk 2]
```diff
@@ -1,4 +1,5 @@
 import { DVCOptions, DVCUser, JSON } from './types'
+import 'react-native-get-random-values'
```
Comment (ajwootto) [inline sdk/js/src/User.ts:2]: will this work in non-react native?

Comment (kaushalkapasi) [reply sdk/js/src/User.ts:2]: yup, this is just a polyfill

Comment (kaushalkapasi) [reply sdk/js/src/User.ts:2]: oh wait.. we already have that library in the root package.json... hmmm

Comment (ajwootto) [reply sdk/js/src/User.ts:2]: we may have it but it wasnt being imported in the SDK

Comment (kaushalkapasi) [reply sdk/js/src/User.ts:2]: will fix imports and update

**Review by ajwootto — APPROVED** [review]

**Pull request merged**